client/client_callback.py

```diff
@@ -22,16 +24,31 @@ def __init__(self, job_data=None, cdn_bucket=None, gogs_url=None):
         self.job = TxJob(job_data)
         self.cdn_bucket = cdn_bucket
         self.gogs_url = gogs_url
+        self.tempDir = tempfile.mkdtemp(suffix="", prefix="client_callback_")
```

Comment:
@rich - OK Changed color settings in IntelliJ so that I will notice the warnings.  Made a ton of fixes.